@navikt/ds-css/stepper/step.css

```diff
@@ -0,0 +1,277 @@
+/* Step */
+.navds-stepper > * .navds-step {
```

Comment:
Denne regelen forstår jeg ikke helt. Er det noen grunn til å skille mellom `.navds-stepper > * .navds-step` og `.navds-step`. Tar jeg feil om klassenavnet ifølge BEM burde vært `.navds-stemmer__step`?

Reply:
Ser ut som det er noen regler som ikke er helt optimalisert etter at horizontal versjon var fikset, så henger igjen fra der. 
Her må vi ta et valg om vi regner Stepper og Step som to forskjellige komponenter, eller som del av en komponent. Hvis vi regner det som en så gir `.navds-stepper__step` mening. Evt siden de stammer fra samme css fil så gir det kanskje mest mening å bruke `.navds-stepper__step`